report_qr/models/ir_actions_report.py

```diff
@@ -0,0 +1,17 @@
+from odoo import api, models
+import qrcode
+import io
+
+
+class IrActionsReport(models.Model):
+    _inherit = 'ir.actions.report'
+
+    @api.model
+    def qr_generate(self, value, box_size=3, border=5, **kwargs):
+        try:
+            qr = qrcode.make(value, box_size=box_size, border=border, **kwargs)
+            arr = io.BytesIO()
+            qr.save(arr, format='png')
```

Comment:
Won't it save to png as default? I'd leave it open so an image_factory (like svg) can be passed to the controller

Reply:
Please see https://github.com/OCA/reporting-engine/pull/324/commits/3c9b5d589eb0bcfafcd910651295eb1cbdde25ea#diff-0dcdf5095cb728e790b48d4cea6d0ed4R16 for better extension